etn-ccis/blui-react-themes
# Add github CI workflow

Fixes BLUI-5563.



#### Changes proposed in this Pull Request:

- add github CI workflow
- remove circleCI config 
- stop building in circleCI
- revert blui colors package back to 3.1.1



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- make small commit to see job run and remove commit
- https://github.com/etn-ccis/blui-react-themes/actions/runs/8484364014



#### Any specific feedback you are looking for?

-
